wicket/src/runner.rs

```diff
@@ -118,89 +123,54 @@ impl Runner {
         loop {
             // unwrap is safe because we always hold onto a UnboundedSender
             let event = self.events_rx.blocking_recv().unwrap();
+            self.recorder.push(&self.state, event.clone());
             match event {
                 Event::Tick => {
+                    // We want to periodically to update the status bar
+                    // By default this is every 1s.
+                    let redraw =
+                        self.state.service_status.advance_all(TICK_INTERVAL);
```

Comment:
Totally. If we tick too frequently, things can get weird because they can get delayed from buffer bloat. It doesn't matter in this case, and we don't currently have this problem, but we may want to eventually check the durations on events to perform animations etc...